docs/podman-rmi.1.md

```diff
@@ -15,10 +15,11 @@ Removes one or more locally stored images.
 
 **-all**, **-a**
 
-Remove all of the locally storage images
-**--force, -f**
+Remove all of the local storage images.
 
-Executing this command will stop all containers that are using the image and remove them from the system
+**--force**, **-f**
+
+Executing this command will stop all containers that are using the image and remove them from the system.
```

Comment:
"This option will cause `podman` to stop all running containers that are using the image before removing them from the system."

Reply:
Not just stop, remove - we can't leave containers around if they refer to images that no longer exist

Reply:
I tweaked both suggestions to:

"This option will cause podman to remove all containers that are using the image before removing the image from the system."

Holler if it needs further tweaking.